docs/ReleaseGuide.md

```diff
@@ -13,21 +13,23 @@ The release of the `python-bindings` repository is made directly from a release
     * `CHANGELOG.md` on `python-bindings-v2.1.1.1`.
     * There is no need to bump the version anywhere else, since we use the [python-versioneer](https://github.com/python-versioneer/python-versioneer/) for maintaining the version everywhere else.
 
-4. [Draft a New Release](https://github.com/precice/python-bindings/releases/new) in the `Releases` section of the repository page in a web browser.
+4. *Optional* test the [py-pyprecice Spack package](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-pyprecice/package.py) using `spack dev-build py-pyprecice@develop`.
```

Comment:
Sounds good to introduce this as a manual step :+1: 

Do we really want to do this as optional? Additionally: This might take quite some time. Maybe this is an argument to just regularly do this step in the CI?

Reply:
It will take substantially longer in the CI though.
A problem with cron jobs is that failure are not easy to discover.

Reply:
We could also solve this with a `workflow_dispatch` that you one can run when releasing.

Reply:
Testing spack is generally a bit a hen-and-egg-problem. If we remove the `package.py` here we could also consider the complete spack package as a downstream dependency: If it breaks after a release we need a bugfix. But that's ok and should not happen too often.

I'm optimistic that with spack relying on `pip` we will not have a lot of trouble in the future. If there is trouble we can still go back and add some test. My conclusion: Let's remove this optional step see how this will turn out.

Reply:
A good reason to manually test a release is that one needs to test the released package anyhow when adding the version to spack.
If the same person who tests during the release also updates the spack package later on, then there will be no rebuild of dependencies necessary.